src/riscv_opcodes/go_utils.py

```diff
@@ -9,12 +8,19 @@
 logging.basicConfig(level=logging.INFO, format="%(levelname)s:: %(message)s")
 
 
-def make_go(instr_dict: InstrDict):
-
-    args = " ".join(sys.argv)
-    prelude = f"""// Code generated by {args}; DO NOT EDIT."""
-
-    prelude += """
+def make_go(instr_dict: InstrDict, extensions: list[str]):
+    comment = """// Code generated with following arguments; DO NOT EDIT.
+/*
+make inst.go"""
+    if extensions:
+        comment += " EXTENSIONS='\\\n"
+        n = 4
+        exts = sorted(extensions)
+        for i in range(0, len(exts), 4):
+            ext_line = " ".join(exts[i : i + n])
```

Comment:
@mengzhuo , This is a bit misleading. The piece of code will end up in breaking the logic intended. The chunk size is controlled by `n` and the iteration is hardcoded as `4` and again inner logic still depends on n. If someone later changes `n` to `8` _(thinking it controls grouping)_, the code breaks because, outer loop still advances by `4`, but inner slice chooses `8`. Consequently, creating duplicated extensions and overlapping groups, leading to generating incorrect outputs.